zap/src/main/java/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/HttpPanelSyntaxHighlightTextArea.java

```diff
@@ -70,7 +70,7 @@
     private static final String FADE_CURRENT_HIGHLIGHT_LINE = "fadehighlightline";
     private static final String SHOW_WHITESPACE_CHARACTERS = "whitespaces";
     private static final String SHOW_NEWLINE_CHARACTERS = "newlines";
-    private static final String MARK_OCCURRENCES = "markocurrences";
+    private static final String MARK_OCCURRENCES = "markoccurrences";
```

Comment:
This should be reverted (or migrate the old configurations).

Reply:
Initially I'll drop, but if you can point me to a similar migrator, I might make a PR for it.

Reply:
That would be done in `Constant` class (similar to the other `upgradeFrom` methods).